addons/mail/static/src/models/message_tracking_value/message_tracking_value.js

```diff
@@ -0,0 +1,193 @@
+/** @odoo-module **/
+
+import { registerNewModel } from '@mail/model/model_core';
+import { attr, many2one } from '@mail/model/model_field';
+
+import { format } from 'web.field_utils';
+
+function factory(dependencies) {
+
+    class MessageTrackingValue extends dependencies['mail.model'] {
+
+        //----------------------------------------------------------------------
+        // Public
+        //----------------------------------------------------------------------
+        /**
+         * @static
+         * @param {Object} data
+         * @return {Object}
+         */
+         static convertData(data) {
+            const data2 = {};
+            if ('id' in data) {
+                data2.id = data.id;
+            }
+            if ('changed_field' in data) {
+                data2.changedField = data.changed_field;
+            }
+            if ('currency_id' in data) {
+                data2.currencyId = data.currency_id;
+            }
+            if ('field_type' in data) {
+                data2.fieldType = data.field_type;
+            }
+            if ('new_value' in data) {
+                data2.newValue = data.new_value;
+            }
+            if ('old_value' in data) {
+                data2.oldValue = data.old_value;
+            }
+            return data2;
+        }
+
+        //----------------------------------------------------------------------
+        // Private
+        //----------------------------------------------------------------------
+
+        /**
+         * @override
+         */
+         static _createRecordLocalId(data) {
+            return `${this.modelName}_${data.id}`;
+        }
+
+        /**
+         * @private
+         * @returns {string}
+         */
+         _computeChangedField() {
+            return _.str.sprintf(this.env._t("%s:"), this.changedField);
+        }
+
+        /**
+         * @private
+         * @param {string} fieldType
+         * @param {string|boolean|float|number|integer|Array|Object} value
+         * @returns {string}
+         */
+        static _formatMessageTrackingValue(fieldType, value) {
```

Comment:
guideline order: static above non-static of the same category, and then alphabetical